compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -316,8 +329,8 @@ define_tables! {
 
 #[derive(Copy, Clone, MetadataEncodable, MetadataDecodable)]
 enum EntryKind {
-    AnonConst(mir::ConstQualifs, Lazy<RenderedConst>),
-    Const(mir::ConstQualifs, Lazy<RenderedConst>),
+    AnonConst,
+    Const,
```

Comment:
Very nice! I wonder if some time soon this whole `enum` could be subsumed by just `DefKind`.